src/components/SwaggerUI.js

```diff
@@ -1,19 +1,15 @@
 import styles from '../index.css';
 
 import React, {useEffect} from 'react';
-// tslint:disable
```

Comment:
This one and line 8 I don't know what they're for.   I don't think we use this component at all anymore.  It was imported in components/AppWithRouterAccess.js  and commented out  {/*<Route path = '/swaggerUI' component={SwaggerComp} />*/}

Reply:
So this is probably okay?

Reply:
I guess :shrug: